src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -90,6 +90,9 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( attributeValue.equals( "0px" ) ) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=47&issues=AWUAdAo6opYCBC6oLI0b&open=AWUAdAo6opYCBC6oLI0b)

[ISSUE_KEY]: AWUAdAo6opYCBC6oLI0b